src/main/java/nextstep/subway/section/domain/Sections.java

```diff
@@ -27,8 +27,11 @@ protected Sections() {
     }
 
     public Sections(List<Section> sections) {
-        firstStationId = sections.get(0).getUpStationId();
-        lastStationId = sections.get(sections.size() - 1).getDownStationId();
+        int firstIdx = 0;
+        firstStationId = sections.get(firstIdx).getUpStationId();
```

Comment:
```suggestion
        this.firstStationId = sections.get(firstIdx).getUpStationId();
```

이렇게 하는게 다른 코드들과의 일관성도 더 있어보일 것 같아요

Reply:
오 다른 메서드들과의 일관성은 고려하지 못했는데, 앞으로 좀 더 신경써야 할 부분으로 생각해야겠군요! 감사합니다.